index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        
+        <link href="https://fonts.googleapis.com/css?family=Oxygen:300,400" rel="stylesheet">
+        <link rel="stylesheet" href="css/style.css">
+        <link rel="shortcut icon" type="image/png" href="img/cherry.png">
+        
+        <title>VueJS Academy &mdash; MAfonina's HTML&amp;CSS homework</title>
+        
+        <link rel="apple-touch-icon" sizes="180x180" href="/favicons/apple-touch-icon.png">
+        <link rel="icon" type="image/png" sizes="32x32" href="/favicons/favicon-32x32.png">
+        <link rel="icon" type="image/png" sizes="16x16" href="/favicons/favicon-16x16.png">
+        <link rel="manifest" href="/favicons/site.webmanifest">
+        <link rel="mask-icon" href="/favicons/safari-pinned-tab.svg" color="#5bbad5">
+        <link rel="shortcut icon" href="/favicons/favicon.ico">
+        <meta name="msapplication-TileColor" content="#da532c">
+        <meta name="msapplication-config" content="/favicons/browserconfig.xml">
+        <meta name="theme-color" content="#ffffff">
+    </head>
+    <body class="container">
+       
+        <div class="logo">
+           <svg class="logo__image">
+               <use xlink:href="img/sprite.svg#cherry"></use>
+           </svg>
+        </div>
+        
+        <header class="header">
+            <nav class="main-navigation">
+                <ul class="main-navigation__list">
+                    <li class="main-navigation__item">Home</li>
+                    <li class="main-navigation__item">About</li>
+                    <li class="main-navigation__item">Gallery</li>
+                </ul>
+            </nav>
+            <span>Log in</span>
+        </header>
+        
+        <aside class="sidebar">
```

Comment:
Very good idea to make the sidebar toggleable!
Here are some suggestions to make it better:
1) You actually don't need to add background as a separate element (you can remove this div). Better set background color and all other styles to .side-navigation
2) Move your checkbox out of .site-navigation to make them siblings (in this case you could set styles to .side-navigation through checkbox)